tests/sentry/api/endpoints/test_scim_user_index.py

```diff
@@ -22,6 +23,10 @@
 }
 
 
+def merge_dictionaries(dict1, dict2):
+    return {**dict1, **dict2}
```

Comment:
Unforunately we don't run python 3.9+ so we can't use `dict1 | dict2` to merge dictionaries and need to use dictionary unpacking

Reply:
`dict1.update(dict2)`?

Reply:
I tried updating the dictionary but it mutates the dictionary so it fails for the tests below